src/meta-srv/src/procedure/region_migration.rs

```diff
@@ -550,6 +556,19 @@ impl Procedure for RegionMigrationProcedure {
                     .inc();
                 ProcedureError::retry_later(e)
             } else {
+                error!(
+                    e;
+                    "Region migration procedure failed, region_id: {}, from_peer: {}, to_peer: {}, 
+                    elapsed: {:?}, downgrade_leader_region_elapsed: {:?}, open_candidate_region_elapsed: {:?}, 
+                    upgrade_candidate_region_elapsed: {:?}",
+                    self.context.region_id(),
+                    self.context.persistent_ctx.from_peer,
+                    self.context.persistent_ctx.to_peer,
+                    self.context.volatile_ctx.operations_elapsed,
+                    self.context.volatile_ctx.downgrade_leader_region_elapsed,
+                    self.context.volatile_ctx.open_candidate_region_elapsed,
+                    self.context.volatile_ctx.upgrade_candidate_region_elapsed
+                );
                 METRIC_META_REGION_MIGRATION_ERROR
```

Comment:
non-blocking: Do we need to report the elapsed time to some metrics?

Reply:
resolved in [65b7c6a](https://github.com/GreptimeTeam/greptimedb/pull/5967/commits/65b7c6a0294e17c6838ec91b8576b18d4f53784e)